src/devices/machine/pit8253.cpp

```diff
@@ -163,6 +186,9 @@ void pit8253_device::device_start()
 {
 }
 
+void ps2_pit_device::device_start()
+{
+}
```

Comment:
Not calling the base `device_start` is usually a bad sign.  You should call it even if it’s empty to make it less error-prone later.  Also, you need to save data members.